sound/soc/sof/ipc.c

```diff
@@ -762,7 +762,8 @@ int snd_sof_dsp_mailbox_init(struct snd_sof_dev *sdev, u32 dspbox,
 }
 EXPORT_SYMBOL(snd_sof_dsp_mailbox_init);
 
-int snd_sof_ipc_valid(struct snd_sof_dev *sdev)
+/* new solution base on see xman_get_fw_version */
```

Comment:
"snd_sof_ipc_valid is only used for older firmware with no extended manifest"

Reply:
outdated